src/Components/Rocket/RocketContainer.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import RocketList from './RocketList';
+import rocket from './rocket.jpg';
+import './Rocket.css';
+
+const RocketContainer = () => {
+  const [rockets] = useState([
+    {
+      id: 1,
+      name: 'Falcon 1',
+      description: 'The Falcom Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+      images: rocket,
+    },
+    {
+      id: 2,
+      name: 'Falcon 2',
+      description: 'The tinj huf Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+      images: rocket,
+    },
+    {
+      id: 3,
+      name: 'Falcon 3',
+      description: 'in Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+      images: rocket,
+    },
+  ]);
+
+  return (
+    <div>
+      <RocketList rockets={rockets} />
+    </div>
```

Comment:
Your RocketList is not correctly displayed on the screen like the template. 
Your project : 
![Screenshot from 2022-07-19 15-51-17](https://user-images.githubusercontent.com/89228743/179798797-1d1486bc-3a11-4599-a679-b39aac40bba1.png)
The template : 
![Screenshot from 2022-07-19 15-59-51](https://user-images.githubusercontent.com/89228743/179799013-9864ee8b-4245-462d-a7a6-8adcbf4dfd11.png)
----
Please you should make sure that you fixed this. Your project layout should be the same compared to the template. 
To help you, instead of wrapping your RocketList component with a div, wrap it with the Container component from react-bootstrap. And make sure you imported the Container component.